### PR TITLE
Temporary remove Max aggregation and add details how to convert Aggregation into Metrics.

### PR DESCRIPTION
**Pull request by bogdandrutu**
For the moment we do not have a clear use for Max aggregation but we are happy to add it when necessary.

Add a table that explain how to convert stats aggregations to metrics.

### CODE REVIEW

File: stats/Export.md [hunk 1]
```diff
@@ -28,3 +27,14 @@ parameter from the `View` and the aggregated data associated with those `columns
...
+### Aggregation to Metrics
+
+| Aggregation  | Measure Type | Metric Type  | Value Type   |
+|--------------|--------------|--------------|--------------|
+| Count        | Any          | CUMULATIVE   | INT64        |
+| Sum          | Double       | CUMULATIVE   | DOUBLE       |
+| Sum          | Int64        | CUMULATIVE   | INT64        |
+| LastValue    | Double       | GAUGE        | DOUBLE       |
+| LastValue    | Int64        | GAUGE        | INT64        |
+| Distribution | Int64        | CUMULATIVE   | DISTRIBUTION |
```
Comment (rakyll) [inline stats/Export.md]: Any

File: stats/Export.md [hunk 2]
```diff
@@ -28,3 +27,14 @@ parameter from the `View` and the aggregated data associated with those `columns
 * `end_time`: a timestamp, describing the end time of the current stats snapshot.
 
 The library SHOULD provide a means of retrieving the ViewData for any registered view in the system.
+
+### Aggregation to Metrics
+
+| Aggregation  | Measure Type | Metric Type  | Value Type   |
+|--------------|--------------|--------------|--------------|
+| Count        | Any          | CUMULATIVE   | INT64        |
```
Comment (sebright) [inline stats/Export.md]: I think "Int64 or Double" would be clearer than "Any".

Comment (bogdandrutu) [reply stats/Export.md]: Done.

File: stats/Export.md (hunk 1 above)
Comment (bogdandrutu) [reply stats/Export.md]: Done.

**Comment by bogdandrutu** [comment]
Specs for https://github.com/census-instrumentation/opencensus-specs/issues/83

**Pull request merged**